pkg/collector/corechecks/cluster/ksm/customresources/cronjob.go

```diff
@@ -179,6 +184,28 @@ func (f *cronjobv1beta1Factory) MetricFamilyGenerators() []generator.FamilyGener
 				}
 			}),
 		),
+		*generator.NewFamilyGeneratorWithStability(
+			"kube_cronjob_status_last_successful_time",
+			"LastSuccessfulTime keeps information of when was the last time the job was completed successfully.",
```

Comment:
```suggestion
			"LastSuccessfulTime indicates the last time the job was completed successfully.",
```

Reply:
This is in fact not a piece of code that we wrote, but something that has been copied from a 3rd party project:
https://github.com/kubernetes/kube-state-metrics/blob/4a585c91beffdbd1378833ce5f102b401e7e57c5/internal/store/cronjob.go#L177
Even if that’s not perfect, I’d rather try to avoid diverging from the upstream project if not needed.

Reply:
Ah yeah, that's totally fine!